### PR TITLE
Filters test - Jackson dependency - entities package

### PR DESCRIPTION
**Pull request by Agustin-Mediotti**

### CODE REVIEW

**Review by eliosfg — APPROVED** [review]
nice!

File: src/main/java/entities/Filter.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+import javax.annotation.processing.Generated;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
```
Comment (adrianovi) [inline src/main/java/entities/Filter.java:14]: I think this is not necessary.

Comment (Agustin-Mediotti) [reply src/main/java/entities/Filter.java:14]: Why?

Comment (Agustin-Mediotti) [reply src/main/java/entities/Filter.java:13]: Removed line 13

**Pull request merged**